src/vlt/src/config/index.ts

```diff
@@ -579,41 +579,44 @@ export class Config {
     const userConfig = xdg.config('vlt.json')
     await this.#maybeLoadConfigFile(userConfig)
 
-    // don't walk up past a folder containing any of these
-    const stops = ['vlt-workspaces.json', '.git']
     // indicators that this *may* be the root, if no .git or workspaces
```

Comment:
This comment should go too, yes? Or does it apply to `lastKnownRoot`?

Reply:
Ya, makes sense to remove it